Tutorial.playground/Contents.swift

```diff
@@ -0,0 +1,178 @@
+//: Introduction
+
+//: The `Aquifer` library decouples stream processing stages from each other so
+//: that you can mix and match diverse stages to produce useful streaming
+//: programs.  If you are a library writer, `Aquifer` lets you package up
+//: streaming components into a reusable interface.  If you are an application
+//: writer, `Aquifer` lets you connect pre-made streaming components with minimal
+//: effort to produce a highly-efficient program that streams data in constant
+//: memory.
+//:
+//: To enforce loose coupling, components can only communicate using two
+//: commands:
+//:
+//: * `yield`: Send output data
+//:
+//: * `await`: Receive input data
+//:
+//: `Aquifer` has four types of components built around these two commands:
+//:
+//: * `Producer`s can only `yield` values and they model streaming sources
+//:
+//: * `Consumer`s can only `await` values and they model streaming sinks
+//:
+//: * `Pipe`s can both `yield` and `await` values and they model stream
+//: transformations
+//:
+//: * `Effect`s can neither `yield` nor `await` and they model non-streaming
+//: components
+//:
+//: You can connect these components together in four separate ways which
+//: parallel the four above types:
+//:
+//: * `for_` handles `yield`s
+//:
+//: * `>~` handles `await`s
+//:
+//: * `>->` handles both `yield`s and `await`s
+//:
+//: * `>>-` handles return values
+//:
+//: As you connect components their types will change to reflect inputs and
+//: outputs that you`ve fused away.  You know that you`re done connecting things
+//: when you get an `Effect`, meaning that you have handled all inputs and
+//: outputs.  You run this final `Effect` to begin streaming.
+
+import Swiftz
+import Aquifer
+
+//: Producers
+
+//: `Producer`s are effectful streams of input.  Specifically, a `Producer` is a
+//: monad transformer that extends any base monad with a new `yield` command.
```

Comment:
This should probably be rephrased, since Swift does not, in fact, support actual monad transformers.